test/producePublicFakeFactors.py

```diff
@@ -3,8 +3,8 @@
 import os
 
 ## Meta-data
-version = '20160425'
-tag     = 'v0.1.3'
+version = '20160503'
+tag     = 'v0.1.4'
```

Comment:
the tag version should be 0.1.3. Since there is no update on the code itself, these fake factors can be read with tag 0.1.3, so no need to create a new tag.